### PR TITLE
added uz_UZ phoneNumber generator and tests

### PR DESCRIPTION
**Pull request by Abdujabbor**

### CODE REVIEW

File: test/Faker/Provider/uz_UZ/PhoneNumberTest.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public function testPhoneFormat()
+    {
+        $generatedPhone = $this->faker->phoneNumber;
+        $pattern = "/\+998 \(9([0|1|3-5|7-9])\) ([\d]{7})/m";
+        $matches = array();
+        $this->assertSame(preg_match($pattern, $generatedPhone, $matches),
+                        1,
+                    'Phone Format is invalid');
+    }
+}
```
Comment (ppelgrims) [inline test/Faker/Provider/uz_UZ/PhoneNumberTest.php]: We're coding against the PSR-2 standard, please add a new line. Use `make sniff test` to run php codesniffer and phpunit.

File: src/Faker/Provider/uz_UZ/PhoneNumber.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (ppelgrims) [inline src/Faker/Provider/uz_UZ/PhoneNumber.php]: Please remove this DocBlock, you'll be listed as contributor once your PR gets merged and every release each PR that got merged is listed as well.

File: test/Faker/Provider/uz_UZ/PhoneNumberTest.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (ppelgrims) [inline test/Faker/Provider/uz_UZ/PhoneNumberTest.php]: See previous DocBlock comment

File: src/Faker/Provider/uz_UZ/PhoneNumber.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * Date: 8/3/18
+ * Time: 1:51 PM
+ */
+
+namespace Faker\Provider\uz_UZ;
+
+class PhoneNumber extends \Faker\Provider\PhoneNumber
+{
+    protected static $formats = array(
+        '+998 (90) #######',
```
Comment (ppelgrims) [inline src/Faker/Provider/uz_UZ/PhoneNumber.php]: Do you have a source for this? Does it comply with the license of this package?

**Comment by pimjansen** [comment]
Bump

**Comment by pimjansen** [comment]
I am going to close this PR since it is stale. Feel free to reopen it if needed later on.
Thanks for contributing!

**Pull request closed without merging**